README.md

```diff
@@ -732,7 +732,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Rwitesh Bera](https://github.com/rwiteshbera)
 
 - ### **S**
-
+  - [Saeem Ahmed](https://github.com/saeem92)
```

Comment:
There's a missing line after the header here.
```suggestion

  - [Saeem Ahmed](https://github.com/saeem92)
```

You can see this at https://raw.githubusercontent.com/saeem92/hacktoberfest-practice/main/README.md
![image](https://user-images.githubusercontent.com/51391473/162628251-c3e103fa-404b-40de-b980-f7bad928913e.png)